src/py/py_multipatch.cpp

```
@@ -602,18 +602,24 @@ void GetInterfaceOrientation(
   // Checks
+  if (para_dim_ < 2) {
+    splinepy::utils::PrintAndThrowError(
+        "Curves are not supported for automatic computatation of interface "
+        "orientations");
+  }
+  if (dim_ < 2) {
+    splinepy::utils::PrintAndThrowError(
+        "Scalar splines are not supported for automatic computation of "
+        "interface orientations");
+  }
   if ((para_dim_ != pyspline_end->SplinepyParaDim())
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)